src/components/authoring/authoring-app.tsx

```diff
@@ -162,6 +164,20 @@ export default class AuthoringApp extends React.Component<IProps, IState> {
                 </div>
               </label>
               <br/>
+              <input
+                type="checkbox"
+                checked={enableStudentLanguageSwitching}
+                data-cy="enableStudentLanguageSwitching"
+                onChange={this.handleEnableStudentLanguageSwitching}
+              />
+              <label>
+                Enable students to switch between languages
+                <div className={css.help}>
+                  When this option is turned on, students will have the option to switch between languages
+                  if there are multiple languages available.
```

Comment:
I'd add a note here about the UI not supporting lots of languages. 2 or 3 translations will be the maximum. The author should check the UI to make sure it is usable.

Reply:
Also this should be clarified that it disables the ability for the teacher to choose whether a student sees a different language in the glossary dashboard.